xml/System.Security.Cryptography.Xml/RSAKeyValue.xml

```diff
@@ -168,7 +168,13 @@
       <Docs>
         <summary>Returns the XML representation of the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</summary>
         <returns>The XML representation of the <see cref="T:System.Security.Cryptography.RSA" /> key clause.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+Based upon https://www.w3.org/TR/xmldsig-core/#sec-RSAKeyValue.
```

Comment:
@rpetrusha How should we show hyperlinks in remarks?

Reply:
If there's a hyperlink with hyperlink text, the format is `[`*hyperlinktext*`](`*url*`)`.
If there's no separate hyperlink text, the format is `<`*url*`>`.

Reply:
Thank you. This is outdated now.